python/src/main/python/pygw/query/statistics/noop_transformer.py

```diff
@@ -0,0 +1,14 @@
+from ..query_result_transformer import QueryResultTransformer
+#from py4j.java_gateway import JavaGateway
+
+
+class NoOpTransformer(QueryResultTransformer):
+    """
+    Transforms Java Long into a pyhton
```

Comment:
This comment seems to be inaccurate.  It actually transforms any primitive java type to the python variant, and if a python variant couldn't be found, it will return the java reference.